cypress/integration/tests/dashboard/add-program/add-program-step-defs.ts

```diff
@@ -24,6 +24,10 @@ And('I enter a program name',()=>{
     addProgramPage.enterProgramName(programName);
 })
 
+And('I enter a program start date',()=>{
+    addProgramPage.enterProgramStartDate("2022-03-07");
```

Comment:
Hi @cheryleigh , can we use current date instead of hardcoding the date value here?